### PR TITLE
Remove handout

### LINKED ISSUE #464: Recommended updates to handouts
Looking at the code reference handouts ([example](https://datacarpentry.org/r-socialsci/instructor/intro-R-handout.html)), I have a few suggestions to improve their usefulness and utilization:

1. Add a note on the handouts to the [Instructor Notes](https://datacarpentry.org/r-socialsci/instructor/instructor-notes.html) main section so people know about them. Be sure to briefly explain what they are and how they should be used
2. Likewise, consider adding inline instructor notes either on the pages they refer to (as an extra resource) and/or pages that rely/build on those lessons (as a "getting up to speed" help)
3. Standardize how you deal with the penguins data and with libraries. Right now, [Starting with Data](https://datacarpentry.org/r-socialsci/instructor/starting-with-data-handout.html) assumes you have the data downloaded locally in a `data` subdirectory but includes other `library()` calls, [Data Wrangling](https://datacarpentry.org/r-socialsci/instructor/data-wrangling-handout.html) loads the penguins data from a library, and [data visualization](https://datacarpentry.org/r-socialsci/instructor/data-visualisation-handout.html) assumes you already have both the data and libraries loaded into memory. My suggestion here is to explicitly include all libraries and the data imports in each handout so that it doesn't matter if you do them sequentially or individually.
4. Modify or remove the paragraph at the start of Data Wrangling and Data Visualization that reads "This document contains all of the functions that we have covered thus far in the course. It will be updated every week, after we’ve added new skills. Each function is presented alongside an example of how it is used." The updated weekly clause is confusing and probably unhelpful. The rest is also unclear, particularly since each handout has only that section and is not cumulative.
5. Consider restructuring this so all the code handouts are on a single page, but in sections. Sections are still linkable, and this would make it easier for learners to see how things are structured. If this is not possible or preferred, it would at least be helpful to rename the handouts with numbers so they appear in order in the menu (e.g. `Code Handout 1`...).

### PR DESCRIPTION
**Pull request by juanfung**
Closes #464 

This PR removes the R handout from both the learners view navigation (via `config.yml`) and from the learners sub-directory.

<details>

For more guidance on how to contribute changes to a Carpentries project, please review [the Contributing Guide](CONTRIBUTING.md) and [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html).

Please keep in mind that lesson Maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum. If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.

</details>

### CODE REVIEW

File: index.md
Comment (jessesadler) [inline index.md:1]: I think the changes here are part of PR #488 and should not be changed here.

File: instructors/instructor-notes.md
Comment (jessesadler) [inline instructors/instructor-notes.md:1]: I think the changes here are part of PR https://github.com/datacarpentry/r-socialsci/pull/488 and should not be changed here.

**Review by jessesadler — COMMENTED** [review]
Looks good to me. Worked as expected. The only change that needs to happen is to remove the changes to index.md and instructors/instructor-noted.md, which are from PR #488.

**Comment by juanfung** [comment]
@jessesadler oh dear...it seems i created the handout branch from the instructor notes branch. sloppy! i will fix. 

so glad you reviewed!

**Comment by juanfung** [comment]
closing due to messy branch

**Pull request closed without merging**